connections/src/app/qr-user-profile/qr-user-profile.component.css

```diff
@@ -0,0 +1,5 @@
+/* Toggle A */
```

Comment:
good use of regular CSS here. 
I will see what we can do with tailwind. or keep it